packages/tooltip/src/widget.ts

```diff
@@ -223,15 +223,23 @@ export class Tooltip extends Widget {
     const style = window.getComputedStyle(this.node);
     const paddingLeft = parseInt(style.paddingLeft!, 10) || 0;
 
+    const host =
+      (editor.host.closest('.jp-MainAreaWidget > .lm-Widget') as HTMLElement) ||
+      editor.host;
+
     // Calculate the geometry of the tooltip.
     HoverBox.setGeometry({
       anchor,
-      host: editor.host,
+      host: host,
```

Comment:
Minor nit-pick for conciseness:

```suggestion
      host,
```